Gruntfile.js

```diff
@@ -0,0 +1,20 @@
+module.exports = function(grunt) {
+   grunt.initConfig({
+       jshint: {
```

Comment:
You have jshint configured but you have no .jshintrc file to drive the behavior, see https://raw.githubusercontent.com/asterisk/node-voicemail-data/master/.jshintrc for an example of how to configure jshint.